java/src/jmri/jmrix/can/cbus/swing/cbusslotmonitor/CbusSlotMonitorDataModel.java

```diff
@@ -55,7 +55,7 @@
     
     static protected final int[] CBUSSLOTMONINITIALCOLS = {0,1,2,4,5,6,9};
     
-    CbusSlotMonitorDataModel(CanSystemConnectionMemo memo, int row, int column) {
+    public CbusSlotMonitorDataModel(CanSystemConnectionMemo memo) {
```

Comment:
It would be good to have a comment that this constructor is public since it's used by scripts. Otherwise, someone might later remove `public` without ever notice the problem.

`// The constructor is public since it's used by user scripts.`

Reply:
Thanks for spotting @danielb987 , I've updated the Constructor Javadoc in latest push to this.